content/articles/geospatial_standards/index.Rmd

```diff
@@ -0,0 +1,92 @@
+
+## Why use open standards?
+
+- Open file standards ease collaboration, portability and compatibility between users, machines and applications.
+- Their (file) structure is fully documented.
+  - Consequently, scientists and programmers can build new software / packages and make innovations that use these standards, while maintaining interoperability with existing applications.
+  - And, it guarantees that your data will still be readable in a hundred years from now, independently of which IT corporation or software is dominant at that time...
+  
+Luckily, quite a list of open standards is available!
+Below, some powerful and widely-used single-file formats are introduced.
+Single-file data sources are readily amenable to exchange and publication.
+
+I see you can't wait to start practicing, so you can also head straight over to the [tutorial on vector formats](../../tutorials/spatial_standards_vector/) and the [tutorial on the GeoTIFF raster format](../../tutorials/spatial_standards_raster/)!
+
+## A few words on the GDAL library
+
+**[GDAL](https://gdal.org)** (Geospatial Data Abstraction Library) is by far the most used collection of open-source drivers for:
+
+- [a lot](https://gdal.org/drivers/raster/index.html) of raster formats;
+- [a lot](https://gdal.org/drivers/vector/index.html) of vector formats.
+
+In other words, it is the preferred workhorse for reading and writing many geospatial file formats, used in the background by [a lot](https://gdal.org/software_using_gdal.html#software-using-gdal) of geospatial applications.
+Using GDAL is the easiest way to conform to open standards.
+
+So, in R we use packages that use GDAL in the background, such as `rgdal`, `sp`, `sf`, `raster` and `stars`.
+
+## The GeoPackage file format
+
+- Its website is <https://www.geopackage.org>.
+- It is a standardized implementation of an SQLite database for geospatial data.
+Hence, a GeoPackage is a **binary** file (`filename.gpkg`).
+It shares this property with shapefiles, which however pose multiple limitations ^[
+Some problems with shapefiles is they're not an open format, they consist of multiple files and they have restrictions regarding file size, column name length, number of columns and the feature types that can be accommodated.
+], 
+so the GeoPackage is a more than suitable replacement.
+- The GeoPackage can store one or _multiple_ **vector** layers (points, lines, polygons and related feature types).
+Beside vector data, it can also store **raster** data^[
+For larger rasters however, the GeoTIFF file format will need less disk space and is therefore recommended.
+]
+or extra standalone **tables**.
+- The GeoPackage standard is [maintained](https://www.opengeospatial.org/standards/geopackage) by the [Open Geospatial Consortium](https://www.opengeospatial.org/) (OGC), which stands out as a reference when it comes to open geospatial standards.
+
+## The GeoJSON file format
```

Comment:
For ArcGis users: this is the equivalent of a .shp file (and it's associated files)

Reply:
Rather, the GeoPackage is the equivalent of the shapefile. Binary, self-contained, exchangeable. That's also how it's connected in literature.

But, it's not a bad idea to more explicitly state this and address ArcGis users indeed.

GeoJSON is really something of its own (text).

Reply:
I wouldn't call GeoPackage the equivalent of a shapefile, because it can contain multiple shape and raster files.

Reply:
That is indeed something extra for GPKG, there you have a point. In that sense, I think they're all different things. Anyhow, you'll find a lot of places where GPKG is seen as a successor to the shapefile (e.g. [1](https://geocompr.robinlovelace.net/read-write.html#file-formats), [2](https://feed.terramonitor.com/shapefile-vs-geopackage-vs-geojson/), [3](https://www.gis-blog.com/geopackage-vs-shapefile/)), but [here](https://carto.com/blog/fgdb-gpkg/) is an interesting one where FGDB is also discussed, next to shapefile & GPKG.

Reply:
The following (from [this source](https://carto.com/blog/fgdb-gpkg/)) could be an important aspect where FGDB differs in terms of portability, with shapefile & GPKG:

> any FGDB data you receive will be a zip file that unzips to a .gdb directory

One can attach files to an email, but FGDB seems to require a folder setup as well.

Reply:
Handled by 890e8a8